app/renderer/src/actions/Session.js

```diff
@@ -662,8 +663,15 @@ export function newSession(caps, attachSessId = null) {
 export function saveSession(server, serverType, caps, params) {
   return async (dispatch) => {
     let {name, uuid} = params;
-    dispatch({type: SAVE_SESSION_REQUESTED});
     let savedSessions = (await getSetting(SAVED_SESSIONS)) || [];
+    let duplicateSession = savedSessions.find(
+      (session) => session.name?.toLowerCase() === name?.toLowerCase(),
```

Comment:
We cannot use optional chaining (Parcel 1 does not support it). Also I think it's fine to only do case-sensitive matching, meaning the `toLowerCase` can be removed, and the optional chaining would not be necessary anyway.

Reply:
@eglitise Makes sense. I have updated the snipped to remove the optional and case sensitive checks.